tests/tracking/test_model_registry.py

```diff
@@ -175,22 +127,15 @@ def verify_pagination(rm_getter_with_token, expected_rms):
             result_rms.extend(result)
         assert [rm.name for rm in expected_rms] == [rm.name for rm in result_rms]
 
-    try:
-        verify_pagination(
-            lambda tok: mlflow_client.search_registered_models(
-                filter_string=filter_string, max_results=max_results, page_token=tok
-            ),
-            filter(filter_func, rms),
-        )
-    except Exception as e:
-        raise e
-    finally:
-        # clean up test
-        for name in names:
-            mlflow_client.delete_registered_model(name)
```

Comment:
Since each test uses a fresh server (which might not be efficient), we no longer need a clean-up like this.